ports/servoshell/Cargo.toml

```diff
@@ -118,7 +119,7 @@ raw-window-handle = "0.6"
 shellwords = "1.0.0"
 surfman = { workspace = true, features = ["sm-x11", "sm-raw-window-handle-06"] }
 tinyfiledialogs = "3.0"
-webxr = { git = "https://github.com/servo/webxr", features = ["ipc", "glwindow", "headless"] }
+webxr = { git = "https://github.com/servo/webxr", features = ["ipc", "glwindow", "headless"], optional = true }
```

Comment:
I'm not convinced that making webxr an optional feature of servoshell is useful. servoshell is a simple test embedding, so nobody should be depending on it. I think servoshell should enable the libservo webxr feature unconditionally.